hello-ih/src/Hello/Landing.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+export default function Landing() {
+  return (
+    <div className="landing-container">
+      <div className="hero">
+
```

Comment:
Se me ocurrío para resolver lo de los atomos que los agregues aquí y modificarlos en el css
```suggestion
<img className="atom" src="./images/react-logo.svg"/>
<img className="atom1" src="./images/react-logo.svg"/>
<img className="atom2" src="./images/react-logo.svg"/>
```